app/components/why-input.jsx

```diff
@@ -0,0 +1,40 @@
+// https://github.com/EnCiv/civil-pursuit/issues/68
+
+'use strict'
+import React from 'react';
+import Point from './point';
+import PointInput from './point-input';
+import { createUseStyles } from 'react-jss';
+
+
+function WhyInput(props) {
+    const { point = { subject: "", description: "", vState: "secondary" }, defaultValue = { subject: "", description: "" }, onDone = () => { }, className, ...otherProps } = props;
+    const classes = useStyles();
+
+    const handleOnDone = ({ valid, value }) => {
+        onDone({ valid, value })
```

Comment:
Okay - here is a new requirement.  the pointObj being created by the whyInput component should add a parentId property to the why point, and it should be the _id of the point.

eg  value.parentId=point._id

also in the test data for stories, make sure there is an _id in each pointObj.   _id is a required field for all mongo documents (meaning objects).